test/router.test.ts

```diff
@@ -0,0 +1,12 @@
+import indefinite from 'indefinite';
+import createRouter, { Routes } from '../src/router';
+
+describe('router', (): void => {
+  const router = createRouter();
+
+  Object.entries(Routes).forEach(([key, value]) => {
+    it(`has ${indefinite(key)} route`, (): void => {
```

Comment:
the scope of these tests should alert of inconsistencies between the `enum` and the routes, so it's helpful to localize that problem